### PR TITLE
feat: add luarocks OPENSSL config

### PR DESCRIPTION
**Pull request by Fuchange**
[apisix/apisix#3469](https://github.com/apache/apisix/issues/3469)

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

File: run.sh [hunk 1]
```diff
@@ -10,6 +10,8 @@ sudo rm -rf /usr/local/apisix/
 rm -rf /tmp/apisix
 
 wget https://github.com/apache/apisix/raw/master/rockspec/apisix-$version-$iteration.rockspec -O apisix-$version-$iteration.rockspec
+sudo luarocks config variables.OPENSSL_LIBDIR /usr/local/openresty/openssl/lib/
+sudo luarocks config variables.OPENSSL_INCDIR /usr/local/openresty/openssl/include
```
Comment (imjoey) [inline run.sh:14]: @Fuchange just for prettier code:

```suggestion
sudo luarocks config variables.OPENSSL_INCDIR /usr/local/openresty/openssl/include/
```

**Review by imjoey — CHANGES_REQUESTED** [review]
@Fuchange please reformat your commit message, staring with `feat:` to make it semantic. Thanks.

File: run.sh (hunk 1 above)
Comment (Fuchange) [reply run.sh]: :ok_hand: done

**Review by imjoey — APPROVED** [review]
/lgtm

Thanks @Fuchange .

**Review by membphis — APPROVED** [review]

**Pull request merged**